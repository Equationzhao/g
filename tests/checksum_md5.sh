output="$(g -term-width 200 --no-config --no-update --icons --checksum -ca md5 tests/test_data )"
echo "$output" | diff - tests/checksum_md5.stdout
