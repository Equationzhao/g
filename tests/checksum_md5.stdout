                                 [0;94m ?.dir[0m                                                      [4ma606ba86e12b9c44156708785c802b8f[0m [0;37m 12.yml[0m                                                     
[4m92cfa388199f79fe428cc2b1c05a7470[0m [0;37m '   space file '[0m                                           [4m32ab7b0c6e5c0847de57dbe972bf41b1[0m [0;93m 13.md[0m                                                      
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.1.112[0m                                                    [4md41d8cd98f00b204e9800998ecf8427e[0m [0;95m 14.gif[0m                                                     
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.1.13[0m                                                     [4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 2.19.1[0m                                                     
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.1.2[0m                                                      [4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 2.9.1[0m                                                      
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.2.10[0m                                                     [4mb8a579ea4b065eb718e6b31d1de147f4[0m [38;5;158m 2.c[0m                                                        
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.2.2[0m                                                      [4m9ebe6a30424ad624782f768041ff7fc4[0m [38;5;158m 3.cpp[0m                                                      
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.3.0[0m                                                      [4md41d8cd98f00b204e9800998ecf8427e[0m [38;5;158m 4.py[0m                                                       
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 0.3.84[0m                                                     [4m370ce925d50d503d69273e59ee502ede[0m [38;5;158m 5.java[0m                                                     
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 1.0.1[0m                                                      [4md41d8cd98f00b204e9800998ecf8427e[0m [0;92m 6.sh[0m                                                       
[4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 1.4.1[0m                                                      [4mc8b1d594f6b0d2552d9eb8675daeeee1[0m [38;5;158m 7.html[0m                                                     
[4mdd7ec931179c4dcb6a8ffb8b8786d20b[0m [38;5;153m 1.txt[0m                                                      [4m9df31b1e5143e99625bd5db3e0b8c64a[0m [38;5;158m 8.css[0m                                                      
[4m8b9cdadd23c79735865c571fec78c01e[0m [0;37m 10.json[0m                                                    [4mf1e3734d68080aa033af2f3e36531077[0m [38;5;158m 9.js[0m                                                       
[4m3293af9a7bcc78e7669ea4403711a0cb[0m [0;37m 100KiB[0m                                                     [4md41d8cd98f00b204e9800998ecf8427e[0m [0;37m 9.js~[0m                                                      
[4md2ba7e2a59827e7dd8cae7a7845a40cd[0m [0;37m 11.xml[0m                                                     [4mdd7ec931179c4dcb6a8ffb8b8786d20b[0m [0;35m 1.txt.link[0m[0;97m => [0m[38;5;153m/Users/equationzhao/g/tests/test_data/1.txt[0m  
